<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookInformation</class>
 <widget class="QWidget" name="BookInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 255, 253);
border-radius:30px;</string>
  </property>
  <widget class="QPushButton" name="pushButton_lend">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>440</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius:10px;
color: rgb(255, 255, 255);
background-color: rgb(255, 179, 160);}
QPushButton:hover{border-radius:10px;
color: rgb(255, 255, 255);
background-color: rgb(255, 104, 93);};
</string>
   </property>
   <property name="text">
    <string>借阅</string>
   </property>
  </widget>
  <widget class="QLabel" name="Titlel_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(76, 190, 34);</string>
   </property>
   <property name="text">
    <string>存本（1/3）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Titlel_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(163, 163, 163);</string>
   </property>
   <property name="text">
    <string>作者：李浩浩</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Titlel_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(8, 69, 81);</string>
   </property>
   <property name="text">
    <string>《老人与手机》</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Titlel_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/666.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>531</width>
     <height>221</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#282828;&quot;&gt;《老人与手机》是作家李浩浩于2021年在长沙写的一篇短篇小说。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#282828;&quot;&gt;该作围绕一位老年的码农，与他的一条电子宠物在他手机中搏斗而展开故事的讲述。尽管李浩浩笔下的老人是悲剧性的，但他身上却有着尼采“超人”的品质，泰然自若地接受失败，沉着勇敢地面对死机，这些“硬汉子”体现了老人的人生哲学和道德理想，即程序员不向bug低头，永不服输的斗士精神和积极向上的乐观人生态度。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Titlel_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(74, 74, 74);</string>
   </property>
   <property name="text">
    <string>3号图书室2架A4002</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>41</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(255, 179, 160);}
QPushButton:hover{border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(255, 104, 93);};
</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QLabel" name="Titlel_x">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(84, 218, 122);</string>
   </property>
   <property name="text">
    <string>借阅成功！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Titlel_xx">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>591</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 89, 89);</string>
   </property>
   <property name="text">
    <string>请取走书籍，并于30天内归还至：三号图书室还书点</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius:10px;
color: rgb(255, 255, 255);
background-color: rgb(255, 179, 160);}
QPushButton:hover{border-radius:10px;
color: rgb(255, 255, 255);
background-color: rgb(255, 104, 93);};
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>BookInformation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>BookInformation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
